apps/designer/app/canvas/shared/breakpoints.ts

```diff
@@ -15,10 +15,12 @@ import {
 
 export const useInitializeBreakpoints = (breakpoints: Array<Breakpoint>) => {
   const [, setCurrentBreakpoints] = useBreakpoints();
-  useEffect(() => {
+  const ref = useRef(false);
+  if (ref.current === false) {
+    ref.current = true;
     setBreakpoints(breakpoints);
     setCurrentBreakpoints(breakpoints);
-  }, [breakpoints, setCurrentBreakpoints]);
+  }
```

Comment:
you are so right, this whole thing inside sdk/stitches/css is written with client-side rendering in mind only, serverside it should always be creating a new instance

Reply:
ok this will work because ref is going to be cleaned and setBreakpoints will be called serverside every time and reset stitches instance and set the new breakpoints, but I am definitely not liking this pattern, we should look how to use context and be able to instantiate things on every render instead of cleaning them up on every render, because that can go sideways easily